pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -85,6 +85,7 @@ func NewCmdRollingUpdate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		Long:    rollingUpdateLong,
 		Example: rollingUpdateExample,
 		Run: func(cmd *cobra.Command, args []string) {
+			printDeprecationWarning(errOut, "rollout", "rolling-update")
```

Comment:
`kubectl rollout` doesn't support ReplicationController. We should probably ask users to use Deployment instead of ReplicationController.

Reply:
Roger that.